src/test/java/pro/sky/HomeWork28/service/EmployeeServiceTest.java

```diff
@@ -0,0 +1,116 @@
+package pro.sky.HomeWork28.service;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pro.sky.HomeWork28.exception.EmployeeAlreadyAddedException;
+import pro.sky.HomeWork28.exception.EmployeeNotFoundException;
+import pro.sky.HomeWork28.exception.EmployeeStorageIsFullException;
+import pro.sky.HomeWork28.model.Employee;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+public class EmployeeServiceTest {
+
+    private final EmployeeServiceImpl employeeServiceImpl = new EmployeeServiceImpl();
```

Comment:
```suggestion
    private final EmployeeService employeeServiceImpl = new EmployeeServiceImpl();
```
надо использовать интерфейсные ссылки